lesson1.tex

```diff
@@ -47,74 +46,57 @@
       
   \section{Procedure}
     \subsection{Setup (10 minutes)}
-      Turn on computers and open the programming environment. Build and turn on bots.
+      Turn on computers and open the programming environment. Build and turn on bots. 
     \subsection{Introduction (5 minutes)}
-      Explain what robots are and connect the lesson to any previously covered class concepts.
+	Start a discussion about robots with the students. Ask what are robots, what are some kinds of robots that they know, and what can robots do. Explain that they will get an opportunity to work with their own robots. 
```

Comment:
"Ask what are robots, the kind of robots they know, and what they know robots can do."